### PR TITLE
Cleanup README.rst:

### PR DESCRIPTION
**Pull request by NickVolynkin**
* Refactor list items.
* Use autonumbering in lists to prevent errors.
* Indent code inside list items to make it belong to those items.
* Use inline-code style for filenames.
* Update link to a pull request with new repo path.
* Fix a few typos and punctuation.
* Use .. code:: bash for proper highlighting
* Use # for comments in bash

### CODE REVIEW

**Comment by NickVolynkin** [comment]
@agjohnson you reacted to the PR in 4 seconds. Are you a bot? :)

**Comment by polyzen** [comment]
@NickVolynkin, this is reStructuredText, not Markdown. Please review the differences between [your version](https://github.com/NickVolynkin/sphinx_rtd_theme/blob/cleanup-readme/README.rst) and [upstream's](https://github.com/rtfd/sphinx_rtd_theme/blob/master/README.rst) (specifically the changes to the note and code [directives](http://docutils.sourceforge.net/docs/ref/rst/directives.html)). Looks good to me, otherwise 👍.

**Comment by NickVolynkin** [comment]
Hi, @polyzen!

> this is reStructuredText, not Markdown

I'm quite well aware of that. :)

> the changes to the note

If you mean that it doesn't render as a Note block – this didn't change. Looks like GitHub just doesn't support this syntax.

> and code directives

Suddenly, GH is less forgiving to wrong indentation than Shpinx which I used to check the output. Fixed it in code blocks throughout the document.

**Comment by NickVolynkin** [comment]
Oh, just a sec, one more change.

**Comment by NickVolynkin** [comment]
> this is reStructuredText, not Markdown

Oh, the commit message! Yep, that's just a habit. Fixed. :)

**Comment by polyzen** [comment]
> Looks like GitHub just doesn't support this syntax.

That's a shame.[The docs](https://ptpb.pw/cpZE.png) don't specify a blank line here -- probably best to inline it.

**Comment by NickVolynkin** [comment]
Most directives in this files have a blank line between declaration and content. But if the docs say so, let it be without a blank line. )

**Comment by NickVolynkin** [comment]
> inline it

Done.

**Comment by NickVolynkin** [comment]
@agjohnson it seems to me that reST indeed requires indent equal to that of the directive. Look what happens if I use 4 and 8 space indents in the list items: https://github.com/NickVolynkin/sphinx_rtd_theme/tree/four-space-indent#set-up-your-environment.

If there's a way to use "normal" 4 space indentation and keep the code formatting, please tell me. It's something that I struggle a lot with.

**Comment by NickVolynkin** [comment]
I'm quite new to reST and now things seem like this to me. Am I wrong? 
```
#. In an ordered list text starts from the fourth character.

   So I should indent next lines with 3 spaces to make them belong to the list.

* In a bulleted list text starts from the third character.

  So next lines should be indented with 2 spaces.

#. Real hell starts

   * When you have a sub-list

     Now it's 3+2 spaces.

     .. code:: bash

        # three more spaces, 8 total.
```

**Comment by polyzen** [comment]
"List item bodies must be left-aligned and indented relative to the bullet .."

4-space indentation must be for directives (possibly just the code directives?).

**Comment by Blendify** [comment]
For one of my projects, we use 3 space indentation. I do not have a strong opinion but I think 3 is correct. @agjohnson can we merge this?

**Comment by agjohnson** [comment]
3 space indentation isn't wrong, as directives just require >3 spaces. We use 4 everwhere else though.

**Comment by agjohnson** [comment]
@NickVolynkin Your example is mostly correct, and @polyzen describes the pattern we use -- list items require special indentation and we use 4 space indentation on all directives. I can comment on the specific blocks in review.

In particular, for a code block nested in a list:

```rst
1. Items can be single lines
2. Items can be multiple,
   dangling lines, but indentation must match (3 chars)
3. Items can contain

   spaces in between lines as long as indentation matches,
   this includes directives:

   .. code::

       # This is 7 character indentation
       # 3 for the list item plus 4 for code
       print(sys.version)
```

File: README.rst [hunk 1]
```diff
@@ -42,23 +42,21 @@ In your ``conf.py`` file:
...
 ``docs/_themes/sphinx_rtd_theme`` then add the following two settings to your Sphinx
-conf.py file:
+``conf.py`` file:
 
 .. code:: python
 
-    html_theme = "sphinx_rtd_theme"
-    html_theme_path = ["_themes", ]
+   html_theme = "sphinx_rtd_theme"
+   html_theme_path = ["_themes", ]
```
Comment (agjohnson) [inline README.rst]: This should be 4 space indent

File: README.rst [hunk 2]
```diff
@@ -74,11 +72,11 @@ file of this repository, and can be defined in your project's ``conf.py`` via
...
-    html_theme_options = {
-        'collapse_navigation': False,
-        'display_version': False,
-        'navigation_depth': 3,
-    }
+   html_theme_options = {
+       'collapse_navigation': False,
+       'display_version': False,
+       'navigation_depth': 3,
+   }
```
Comment (agjohnson) [inline README.rst]: 4 space indent here

File: README.rst [hunk 3]
```diff
@@ -217,45 +214,45 @@ of a front-end developer (vs. that of a python developer). That means installing
...
 -----------------------
 
-1. Install sphinx_ into a virtual environment.
+#. Install sphinx_ into a virtual environment.
 
-.. code::
+   .. code:: bash
 
-    pip install sphinx
+      pip install sphinx
```
Comment (agjohnson) [inline README.rst]: This code block would be 7 space indent (3 for list item, 4 for code)

File: README.rst [hunk 4]
```diff
@@ -217,45 +214,45 @@ of a front-end developer (vs. that of a python developer). That means installing
...
 
-2. Install sass
+#. Install sass.
 
-.. code::
+   .. code:: bash
 
-    gem install sass
+      gem install sass
```
Comment (agjohnson) [inline README.rst]: 7 space indent here

File: README.rst [hunk 5]
```diff
@@ -217,45 +214,45 @@ of a front-end developer (vs. that of a python developer). That means installing
...
 
-    // Install bower and grunt
-    npm install -g bower grunt-cli
+      # Install bower and grunt
+      npm install -g bower grunt-cli
 
-    // Now that everything is installed, let's install the theme dependecies.
-    npm install
+      # Now that everything is installed, let's install the theme dependencies.
+      npm install
```
Comment (agjohnson) [inline README.rst]: 7 space indent here

**Comment by NickVolynkin** [comment]
@agjohnson thanks for explaining. Will update the MR next week. :)

**Comment by Blendify** [comment]
This pull request became inactive and #418 fixes the indent issue so that was merged instead.  Nick Volynkin, please still free to send more PRs in the future.

**Pull request closed without merging**